packages/jest-cli/src/TestRunner.js

```diff
@@ -439,14 +443,13 @@ class ReporterDispatcher {
     );
   }
 
-  onTestResult(config, testResult, results) {
+  onTestResult(test, testResult, results) {
```

Comment:
Ok, sorry about that, I'm checking it through GH, so I don't know if Flow infers something :D